pkg/minikube/mustload/mustload.go

```diff
@@ -98,6 +98,7 @@ func Running(name string) ClusterController {
 
 	if hs == state.None.String() {
 		out.Step(style.Shrug, `The control plane node "{{.name}}" does not exist.`, out.V{"name": cp.Name})
+		out.Step(style.Shrug, `Right-click the PowerShell icon and select Run as Administrator to open PowerShell in elevated mode.`)
```

Comment:
this doesn't seem right, this warning is being printed for anytime any driver on any platform (mac, linux) has a state of none. this advice is not applicable to everyone

Reply:
> this doesn't seem right, this warning is being printed for anytime any driver on any platform (mac, linux) has a state of none. this advice is not applicable to everyone

driver hyperv is not supported for linux system, so this case won't be called forever.
![image](https://user-images.githubusercontent.com/47560998/100685883-2e606380-33b8-11eb-9cb9-b2849fd3b37a.png)

Reply:
I also think that this is the wrong spot for this check, as it seems like it's originating from Line 66 in the logs:

https://github.com/kubernetes/minikube/issues/9589#issue-733490279

Could you please take another look at this?

Reply:
I agree this is not the right place to check for it

Reply:
this check could be in the cmd package.